src/Starcounter.VS.CSBoilerplateApplicationProjectTemplate/MyTemplate.vstemplate

```diff
@@ -0,0 +1,46 @@
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
+  <TemplateData>
+    <Name>Starcounter Boilerplate Application</Name>
+    <Description>A boilerplate project for creating a C# application that executes inside Starcounter</Description>
```

Comment:
Can we do better here? Something that better distinguish this one from the "minimal" Starcounter Application.